vision.md

```diff
@@ -1,52 +1,41 @@
-Stated simply...
+## Stated simply...
 
-**NUnit 3 is a world-class platform for individuals and teams using a test-based approach to .NET software development.**
+**NUnit aims to be a best in breed test framework that enables end to end testing of .NET projects by individuals and teams. It is unopinionated and supports a number of different kinds of tests. NUnit allows for alternative approaches to testing and is usable by team members in a variety of roles.**
 
```

Comment:
What's the intended different meaning between "supports a number of different kinds of tests" and "allows for alternative approaches to testing"?